pkg/stack/unwind/plan_table.go

```diff
@@ -116,14 +119,34 @@ func (ptb *PlanTableBuilder) PlanTableForPid(pid int) (PlanTable, error) {
 			continue
 		}
 
-		res = append(res, buildTable(fdes))
+		res = append(res, BuildTable(fdes))
 	}
 
 	// TODO(kakkoyun): Merge and order instructions of PlanTables.
 	// TODO(kakkoyun): Figure out mapping start address usage.
 	return res[0], nil
 }
 
+// PrintTable is a debugging helper that prints the unwinding table to stdout.
+// TODO(javierhonduco): accept a writer instead of directly printing to stdout.
+func (ptb *PlanTableBuilder) PrintTable(writer io.Writer, path string, start uint64, filterNops bool) error {
```

Comment:
Do we need the `start` as a parameter?

Reply:
Right now no, usually I wanna print the whole thing, thought we could add it as a parameter if necessary, but will remove it in the meantime